### PR TITLE
Build cleanup

### PR DESCRIPTION
**Pull request by sergej-koscejev**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -86,10 +86,6 @@ ext.additionalPomInfo = {
 }
 
 subprojects {
-    // required plugins
-    apply plugin: "base"
```
Comment (alexanderpann) [inline build.gradle:90]: Does this mean that the message about local build detected, now also only shows once and not for every subproject?

File: subprojects/com.mbeddr/build.gradle [hunk 2]
```diff
@@ -130,12 +135,6 @@ private static void configureRepositories(Project project) {
         }
     }
     project.repositories {
-        def disableMavenLocal = project.hasProperty("disableMavenLocal") && project.getProperty("disableMavenLocal").toBoolean()
-        if (!disableMavenLocal) {
-            // we don't use mavenLocal() repo, since it can cause various issues with resolving dependencies,
-            // see https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:case-for-maven-local
-            mavenLocal()
-        }
```
Comment (alexanderpann) [inline subprojects/com.mbeddr/build.gradle:138]: I wonder if the reason for that flag was, that JetBrains needed it for their build. Before removing it, I would check this first but. We don't have it anymore in MPS-Extension though.

File: build.gradle (hunk 1 above)
Comment (sergej-koscejev) [reply build.gradle:90]: I don’t think so, that’s future work.

File: subprojects/com.mbeddr/build.gradle (hunk 2 above)
Comment (sergej-koscejev) [reply subprojects/com.mbeddr/build.gradle:138]: They are not using it in the mbeddr build, see https://teamcity.jetbrains.com/buildConfiguration/MPS_20241_Distribution_TestMbeddrBuild?buildTypeTab=buildTypeSettings#all-projects

Comment (alexanderpann) [reply subprojects/com.mbeddr/build.gradle:138]: This was the reason it was re-introduced: https://github.com/mbeddr/mbeddr.core/commit/85e3519b03ffe63c2520967525cb52a858539727

**Review by alexanderpann — APPROVED** [review]

File: subprojects/com.mbeddr/build.gradle (hunk 2 above)
Comment (sergej-koscejev) [reply subprojects/com.mbeddr/build.gradle:138]: Ah, okay, thanks, I’ll have another look at this.

Comment (sergej-koscejev) [reply subprojects/com.mbeddr/build.gradle]: I removed the relevant commit as I want to make the build somehow do the right thing automatically without extra flags.

**Pull request merged**